pyproject.toml

```diff
@@ -11,7 +11,7 @@ known_first_party = ["pbench"]    # separate "pbench" section
 multi_line_output = 3             # "hanging" indent with dedented paren
 force_sort_within_sections = true # don't separate import vs from
 order_by_type = false             # sort alphabetic regardless of case
-skip = ["fio-histo-log-pctiles.py", "web-server/v0.3/demo.py"] # like black
+extend_skip = ["fio-histo-log-pctiles.py", "web-server/v0.3/demo.py"] # like black
```

Comment:
Thanks; the `skip` vs `extend_skip` semantic wasn't clear in the documentation, and in particular there was no obvious reference to a default `skip` that I'd be overriding instead of extending. Sigh. 😦